### PR TITLE
small corrections; Web is capitalized, webroot is one lowercase word

### PR DESCRIPTION
**Pull request by carlaschroder**
A few small tweaks to align with the style guide. (Yes, there is a style guide!)

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -1011,7 +1011,8 @@
 ),
 
 /**
- * sqlite3 journal mode can be specified using this config parameter - can be
+ * sqlite3 journal mode can be specified using this configuration parameter - 
+can be
```
Comment (DeepDiver1975) [inline config/config.sample.php]: this line looks wrong

Comment (nickvergessen) [inline config/config.sample.php]: Fixed

**Comment by nickvergessen** [comment]
:+1:

**Comment by PVince81** [comment]
:+1:

**Pull request merged**